### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/kind/commit/d5c8a846842b3b3396a327193a45733d21addc06#commitcomment-45717247

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/kind/commit/737fe3681b16b46d28652b50cc9d0f605c946918

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

**Comment by aojea** [comment]
/ok-to-test
/lgtm
/assign @BenTheElder 
Thanks

File: pkg/build/nodeimage/build_impl.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func (c *buildContext) Build() (err error) {
 func (c *buildContext) buildImage(bits kube.Bits) error {
 	// create build container
 	// NOTE: we are using docker run + docker commit so we can install
-	// debians without permanently copying them into the image.
+	// debian's without permanently copying them into the image.
```
Comment (BenTheElder) [inline pkg/build/nodeimage/build_impl.go:58]: this is actually wrong I think, we're pluralizing "debian" as in "debian package", it's not a contraction or possessive.

**Review by BenTheElder — COMMENTED** [review]
thanks, one nit!

sorry for the delay, have been out etc.

File: pkg/build/nodeimage/build_impl.go (hunk 1 above)
Comment (BenTheElder) [reply pkg/build/nodeimage/build_impl.go:58]: it would probably be clearer to instead say "[...] install debian packages [...]"

Comment (jsoref) [reply pkg/build/nodeimage/build_impl.go]: https://github.com/kubernetes-sigs/kind/pull/1997/commits/bd46991731fa429cf8f5be1f2828b014080b1123

**Comment by jsoref** [comment]
@BenTheElder: I'm going to ignore the test failure and hope someone else can address the flakiness.

**Comment by BenTheElder** [comment]
> {"component":"entrypoint","file":"prow/entrypoint/run.go:165","func":"k8s.io/test-infra/prow/entrypoint.Options.ExecuteProcess","level":"error","msg":"Process did not finish before 2h0m0s timeout","severity":"error","time":"2021-01-20T02:32:35Z"}


looks like we got unlucky on the CI and hit an overloaded node.
kubernetes/kubernetes blocking jobs are on a new community owned nodepool where all jobs have strict resource requests and limits, unfortunately we're not, so the CI environment is much noisier.

I'm confident in merging this without the go build passing anyhow, and I have the ACLs to do that here in this repo 🙃

**Comment by BenTheElder** [comment]
/lgtm
/approve
thanks!

**Pull request merged**